strings/reverse_words.py

```diff
@@ -0,0 +1,21 @@
+# Created by sarathkaul on 18/11/19
+
+
+def reverse_words(input_str: str) -> str:
+    """
+    Reverses words in a given string
+    >>> sentence = "I love Python"
+    >>> reverse_words(sentence) == " ".join(sentence.split()[::-1])
```

Comment:
Let's also keep the other test to help the reader understand...
```python
>>> reverse_words(sentence)
'Python love I'
```

Reply:
Okay Cool